src/components/tabs/tabsPaginationService.spec.js

```diff
@@ -1,6 +1,6 @@
 describe('MdTabsPaginationService', function() {
 
-  const TAB_WIDTH = 100;
```

Comment:
nice try at ES2015 support.

You should configure babel properly and use let/const for everything if you're going to use them